app/components/cases/mandatee-selector-panel.hbs

```diff
@@ -0,0 +1,45 @@
+ <Auk::Panel class="auk-u-mb-2">
+        <Auk::Panel::Header
+          @size="auto"
+          class="au-u-background-gray-100"
+        >
+          <Auk::Toolbar @auto={{true}} as |Toolbar|>
+            <Toolbar.Group @position="left">
+              <h4 class="auk-toolbar__title">
+                {{t "ministers"}}
+              </h4>
+            </Toolbar.Group>
+          </Auk::Toolbar>
+        </Auk::Panel::Header>
+        <Auk::Panel::Body>
+          <Auk::CheckboxTree
+            @label='Alle ministers'
+            @items={{this.currentMinisters}}
+            @itemStructure={{hash label="fullName"}}
+            @selectedItems={{this.selectedCurrentMinisters}}
+            @didUpdate={{this.onChangeSelectedMinisters}}
+            @layout="grid"
+          />
+          {{#if this.selectedCurrentMinisters}}
+          <AuHr/>
+          {{/if}}
+          <AuList
+            @divider={{true}}
+            as |Item|
+          >
+            {{#each this.selectedCurrentMinisters as |minister index|}}
```

Comment:
I believe the list is sorted based on priority?

Reply:
The bottom list is just "order of selection":

![image](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/assets/22411874/9d2d5a9b-7a64-4012-92d3-77e504d1cecc)


This may be okay though, perhaps it's not important here.

Reply:
fixed